### PR TITLE
SEO-18702-ASP.NET-Gantt-Exporting-Description-too-long

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Title | Description
-- | --
|Task Category | Site Audite|
|Content Task Link/Mail Screenshot | NA|
|Hotfix |https://github.com/syncfusion-content/asp.netwebforms-docs/pull/520|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187172|
|Work link |https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7BCE6E5440-7D4E-43FE-B673-E79BCD0BF05D%7D&file=Faith-Site%20Audit.xlsx&action=default&mobileredirect=true&wdOrigin=TEAMS-MAGLEV.p2p_ns.rwc&wdExp=TEAMS-TREATMENT&wdhostclicktime=1722849150540&web=1|

Title | Description
-- | --
|Task Category | Site Audite|
|Content Task Link/Mail Scr
Changes made | Reason for change
-- | --
| Page with redirect| To increase page performance |

### CODE REVIEW

File: aspnet/Gantt/Exporting.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
 title: Exporting| Gantt | ASP.NET | Syncfusion
-description: exporting
+description: Learn here about Exporting with the Syncfusion ASP.NET Web Forms Gantt control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:4]: Change description - Learn here about exporting support in Syncfusion ASP.NET Web Forms Gantt control, its elements, and more.

File: aspnet/Gantt/Exporting.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 platform: aspnet
 control: Gantt
 documentation: ug
 ---
-# Export
+# Exporting Sf ASP.NET Web Forms Gantt
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:9]: Updated proper H1

File: aspnet/Gantt/Exporting.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ The below code snippet explains the above behavior,
 {% endtabs %} 
 
 The below screen shot shows Gantt with excel and PDF exporting enabled.
-![](Export_images/Export_img1.png)
+![Export in ASP.NET Web Forms Gantt Exporting](Export_images/Export_img1.png)
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:59]: Updated proper Alt

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/Gantt/Exporting.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 layout: post
 title: Exporting| Gantt | ASP.NET | Syncfusion
-description: exporting
+description: Learn here about exporting support in Syncfusion ASP.NET Web Forms Gantt control, its elements, and more.
 platform: aspnet
 control: Gantt
 documentation: ug
 ---
-# Export
+# ASP.NET Gantt-Invokes export public method of Gantt object to export.
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:9]: Update H1 as Exporting in ASP.NET Web Forms Gantt Control

File: aspnet/Gantt/Exporting.md [hunk 5]
```diff
@@ -56,7 +56,7 @@ The below code snippet explains the above behavior,
 {% endtabs %} 
 
 The below screen shot shows Gantt with excel and PDF exporting enabled.
-![](Export_images/Export_img1.png)
+![PDF exporting enabled in ASP.NET Web Forms Gantt Exporting](Export_images/Export_img1.png)
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:59]: Update proper Alt

File: aspnet/Gantt/Exporting.md [hunk 6]
```diff
@@ -56,7 +56,7 @@ The below code snippet explains the above behavior,
 {% endtabs %} 
 
 The below screen shot shows Gantt with excel and PDF exporting enabled.
-![](Export_images/Export_img1.png)
+![PDF exporting enabled in ASP.NET Web Forms Gantt](Export_images/Export_img1.png)
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:59]: Rename image also

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/Gantt/Exporting.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
 layout: post
 title: Exporting| Gantt | ASP.NET | Syncfusion
-description: exporting
+description: Learn here about exporting support in Syncfusion ASP.NET Web Forms Gantt control, its elements, and more.
 platform: aspnet
 control: Gantt
 documentation: ug
 ---
-# Export
+# Exporting in ASP.NET Web Forms Gantt Control.
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:9]: remove full stop

File: aspnet/Gantt/Exporting.md [hunk 8]
```diff
@@ -56,7 +56,7 @@ The below code snippet explains the above behavior,
 {% endtabs %} 
 
 The below screen shot shows Gantt with excel and PDF exporting enabled.
-![](Export_images/Export_img1.png)
+![Exporting in ASP.NET Web Forms Gantt Control](Export_images/Export_img1.png)
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:59]: Added dot at the end of image Alt

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Comment by Faith-Atieno** [comment]
image

File: aspnet/Gantt/Exporting.md [hunk 9]
```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
-title: Exporting| Gantt | ASP.NET | Syncfusion
-description: exporting
+title: Exporting in ASP.NET Webforms Gantt Control | Syncfusion
+description: Learn here all about exporting support in Syncfusion Essential ASP.NET Webforms Gantt Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/Gantt/Exporting.md:4]: Use ASP.NET Web Forms instead of ASP.NET Webforms

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**